.github/workflows/nodejs.yml

```diff
@@ -44,3 +45,65 @@ jobs:
         run: npm run coverage
         env:
           MONGO_VERSION: ${{ matrix.mongodb-version }}
+
+  build:
+    name: Build docker image
+    needs: test
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') || github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        
+      - name: Configure docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/mia-platform/crud-service
+            docker.io/miaplatform/crud-service
```

Comment:
Is it correct that there is no dash (`-`) here in the label?

Reply:
Unfortunately, it is.
The organization account is called [miaplatform](https://hub.docker.com/u/miaplatform), so I assume it is correct to use this label.